customer-service/src/main/java/com/spaghettininjas/yaposs/controller/CustomerController.java

```diff
@@ -0,0 +1,37 @@
+package com.spaghettininjas.yaposs.controller;
+
+import com.spaghettininjas.yaposs.service.CustomersService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.spaghettininjas.yaposs.repository.entity.Customer;
+import java.util.Arrays;
+
+@RestController
+@RequestMapping("/api/Customers")
+public class CustomerController {
+
+    @Autowired
+    private CustomersService service;
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET, produces=MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Fails [this](https://rules.sonarsource.com/java/RSPEC-4488/) rule.

Reply:
Fixed according to https://rules.sonarsource.com/java/RSPEC-4488/